### PR TITLE
Fix typo in hint::black_box docs

### PR DESCRIPTION
**Pull request by lilyyy411**
situ, -> situations

### CODE REVIEW

**Comment by Dylan-DPC** [comment]
Thanks for the contribution.
However, this looks like it isn't a typo:
»In situ« is a phrase which means »(situated)in the original place«

**Comment by danielhenrymantilla** [comment]
I do agree _in situ_ fits quite oddly here, though.

> would _actually_ be used, where ...

That seems to be, _a priori_, the intent the original author had (given the comma they wrote), and using _in situ_/"in the [very] place" is convoluted at best. By using "actually", it will be English rather than Latin, and in this case, simple English, which is _a fortiori_ more accessible.

_Ergo_, removing that usage of _in situ_ is an improvement, _ceteris paribus_: the author going and submitting a PR, _motu propio_, to discuss it, was thus the right move 🙃

**Comment by 4gboframram** [comment]
I thought that this is what the original author intended but apparently I can't read

File: library/core/src/hint.rs [hunk 1]
```diff
@@ -287,7 +287,7 @@ pub fn spin_loop() {
 /// - Treats the call to `contains` and its result as volatile: the body of `benchmark` cannot
 ///   optimize this away
 ///
-/// This makes our benchmark much more realistic to how the function would be used in situ, where
+/// This makes our benchmark much more realistic to how the function would be used in situations where
```
Comment (ChrisDenton) [inline library/core/src/hint.rs:290]: @4gboframram Would you be willing to make the change suggested by @danielhenrymantilla?
```suggestion
/// This makes our benchmark much more realistic to how the function would actually be used, where
```

**Review by tgross35 — APPROVED** [review]
I'm the one who originally wrote this and meant "in situ" as "in its natural place"/"in position", i.e., how you would come across this function when it's used in real code. But I see that apparently came off a bit abstract, so might as well KISS 🙂.

This definitely isn't a two commit kind of update though, so please squash and correct the commit message @4gboframram

**Comment by Dylan-DPC** [comment]
There hasn't been much interest in this and I think the change doesn't clarify the wording more than the original. Closing this as it's been inactive for a while

**Pull request closed without merging**